### ISSUE TITLE
White band below footer

### ISSUE THREAD

**Issue by simon1066** [zencart/documentation#915]
Although more noticeable on large window sizes a white band has appeared below the footer. I think this was introduced with the addition of the footer's ADA/WCAG link. I'll try to take a look once I've finished with the foldable sidebar menu.

**Comment by dbltoe** [comment]
happens in full screen mode only with Chrome.  Not visible in Mobile.

**Comment by simon1066** [comment]
Also in Firefox.  Not sure about the first option, prefer the same background colour as the rest of the footer. Couldn't get the second option to work though.

**Comment by dbltoe** [comment]
Works from inspect but not "in the wild" using Stylus.

Even tried body '{bsackground-color:#403f4c;}'  but it only works from inspect (F12) as well.

Can't find it in the CSS we changed before or in any of the stylesheets at assets/scss/

**Comment by simon1066** [comment]
I think that removing the `min-height` (two occurrences) from the `footer` style should do it . I'll test a bit more and PR it.

**Comment by scottcwilson** [comment]
I noticed this too.  Thanks for fixing, guys.
